src/modules/ipc.cpp

```diff
@@ -59,11 +60,33 @@ namespace modules {
     for (auto& hook : m_hooks) {
       hook->command = pid_token(hook->command);
     }
-    m_formatter->add(DEFAULT_FORMAT, TAG_OUTPUT, {TAG_OUTPUT});
 
+    m_formatter->add(DEFAULT_FORMAT, TAG_LABEL, {TAG_LABEL});
+    m_formatter->add_optional("output", {});
```

Comment:
This adds a new format, the ipc module never had a format named `output`.

The deprecation here is slightly different to the text module. In the text module we deprecated the `output` format, here we need to deprecate the `<output>` tag on `format`.